src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.figures.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name().toLowerCase();
```

Comment:
```suggestion
        return Color.values()[index].getColor() ;
```
you can use getColor() instead